src/actions/authActions.js

```diff
@@ -45,6 +46,20 @@ export const loginUser = userData => (dispatch) => {
       });
     });
 };
+
+export const logoutUser = userData => (dispatch) => {
+  localStorage.removeItem('jwtToken');
+  localStorage.removeItem('user');
+  return axios
+    .post('/auth/logout', userData)
+    .then(() => {
+    })
+    .catch(error => dispatch({
+      type: GET_ERRORS,
```

Comment:
Can you please extract this into a callable function so it follows the standard of what we've been working with